CONFIGURATION_OPTIONS.md

```diff
@@ -5,11 +5,25 @@ This field does not have the default value. If it is not set, the appId calculat
 Example:
 ```javascript
 {
-  appId:"a-00xx"
+  appId: "a-00xx"
+}
+```
+#### `distributorId` [Optional]
+This parameter is an identifier for distributors.
+`distributorId` does not have a default value and must be set in case of distributors scripts.
+Note: The distributor script has `globalVarName` property that enables to:
+ * initialize LiveConnect configuration for distributors including the `distributorId`.
+ * send distributors events.
+
+If `distributorId` is provided, the `appId` configuration parameter must not be set.
+Example:
```

Comment:
`globalVarName` is independent of `distributorId`. It just allows you to define an arbitrary name for the variable we attach to `window` and that can be used to interact with LiveConnect.

Reply:
I adjust it and removed that, 
but for this:
https://github.com/LiveIntent/live-connect/pull/85#discussion_r999023686
 where should we define the `globalVarName` ?

Reply:
It is an ordinary configuration parameter. Just put the description of it into CONFIGURATION_OPTIONS.md.